### PR TITLE
docs: javadoc aggregation and delombok

### PR DESCRIPTION
**Pull request by stachu540**
### Prerequisites for Code Changes
* [x] This pull request follows the code style of the project
* [x] I have tested this feature

### Changes Proposed
- adding 'io.freefair.lombok' version '5.2.1'
- append all javadocs source to delombok task - [reference](https://docs.freefair.io/gradle-plugins/5.2.1/reference/#_io_freefair_lombok)
- ignores auto-generate 'lombok.config' for each existed project
- adding 'aggregateJavadoc' to generate combined javadoc for new incoming website

### Additional Information 
@iProdigy request it and he have it. Feel free to checking changes. Also i've added new task which he generate whole project aggregate into one javadoc. For new website purpose (soon™️). Here is diffrence between non-delomboked and delomboked files:

diff

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,6 +36,12 @@ subprojects {
 	apply plugin: 'maven-publish'
 	apply plugin: 'com.jfrog.bintray'
 	apply plugin: 'com.jfrog.artifactory'
+	apply plugin: 'io.freefair.lombok'
+	lombok {
+		version = "1.18.10"
```
Comment (iProdigy) [inline build.gradle:41]: Would `1.18.16` be compatible? #220 won't work without it

Comment (stachu540) [reply build.gradle:41]: Yikes? - on the way

File: build.gradle [hunk 2]
```diff
@@ -100,10 +107,11 @@ subprojects {
 	plugins.withType(JavaPlugin) {
 		dependencies {
 			// Annotation processors
-			compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.10'
-			annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.10'
-			testCompileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.10'
-			testAnnotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.10'
+			// DEPRECATED: removal if 'io.freefair.lombok' has been applied
```
Comment (PhilippHeuer) [inline build.gradle:110]: Can you remove those comments?

**Review by PhilippHeuer — CHANGES_REQUESTED** [review]
Looks good - i think this can be merged after you remove the outcommented lombok dependencies.

File: build.gradle (hunk 2 above)
Comment (stachu540) [reply build.gradle]: done

**Review by PhilippHeuer — APPROVED** [review]

**Pull request merged**